services/processor/native/sanitize_code.go

```diff
@@ -27,7 +34,40 @@ func (s *service) createSanitizer() *sanitizer.Sanitizer {
 			`"github.com/orbs-network/orbs-contract-sdk/go/sdk/v1/safemath"`: true,
 			`"github.com/orbs-network/orbs-contract-sdk/go/sdk/v1/service"`:  true,
 			`"github.com/orbs-network/orbs-contract-sdk/go/sdk/v1/state"`:    true,
+
+			// Text
+			`"fmt"`:           true,
```

Comment:
wait wait wait
we can't whitelist anything that might result in non-deterministic transactions
specifically, @talkol has objected to whitelisting `fmt` for these reasons

Can you please explain next to each import why it's whitelisted? preferably, create a struct:
```
type whitelist struct {
   packageName string
   useCase string
```
then populate it with justifications for whitelisting stuff

Reply:
Instead of struct I replaced `bool` with a string.